requirements/openvino.txt

```diff
@@ -4,4 +4,4 @@ nncf==2.3.0
 onnx==1.10.1
 openmodelzoo-modelapi==2022.2.0
 openvino==2022.2.0
-openvino-dev==2022.2.0
+openvino-dev[tensorflow2,mxnet,caffe]==2022.2.0
```

Comment:
Just curiosity, above 3 all needed for download feature?

Reply:
Yes, indeed. Please refer the **Implementation** column of the [list](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md#classification-models). Note that **paddle**  library will be required as well if we wanted to support it.